app/src/main/java/com/example/finalapplication/screen/chatroom/ChatRoomActivity.kt

```diff
@@ -43,12 +48,21 @@ class ChatRoomActivity : BaseActivity<ActivityChatRoomBinding>(ActivityChatRoomB
                     getNewid().toString(),
                     Message.chatbox,
                     System.currentTimeMillis(),
-                    currentUser.id.toString(),
+                    currentUser?.id.toString(),
                     adversaryUser?.id.toString()
                 )
                 message.image = uri.toString()
-                message.roomId = currentUser.id + adversaryUser?.id
-                adversaryUser?.let { it1 -> chatViewModel.sendMessage(currentUser, it1, message) }
+                message.roomId = currentUser?.id + adversaryUser?.id
+                isLoadImage = true
+                adversaryUser?.let { it1 ->
```

Comment:
```suggestion
                adversaryUser?.let { user ->
```
ko chơi it1 ,it2 đâu nhé